features/step_definitions/user_can_create_account_steps.rb

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
rename this file to `basic_steps` instead

Reply:
Resolved, name changed